lib/compiler-singlepass/src/codegen_x64.rs

```diff
@@ -441,7 +441,7 @@ impl<'a> FuncGen<'a> {
                     Size::S64,
                     Location::Memory(
                         Machine::get_vmctx_reg(),
-                        self.vmoffsets.vmctx_gas_limiter_pointer() as i32,
+                        self.vmoffsets.vmctx_gas_limiter_pointer_begin() as i32,
```

Comment:
Don't think this particular rename makes much sense.

Reply:
This change was primarily based on the pre-existing methods on VmOffsets – all of the methods returning an offset to a field had a `_begin` suffix, so I figured that consistency for these two fields is probably the best.

Reply:
I don't think naming matters too much in the grand scheme of things, but to me it also seems that `_begin` terminology is consistent with what we have elsewhere. Compare, for example, `vmctx_trap_handler_begin`.